exasol/nb_connector/extension_wrapper_common.py

```diff
@@ -43,16 +46,17 @@ def encapsulate_bucketfs_credentials(
     """
 
     bfs_host = conf.get(CKey.bfs_host_name, conf.get(CKey.db_host_name))
-    # For now, just use the http. Once the exasol.bucketfs is capable of using
-    # the https without validating the server certificate choose between the
-    # http and https depending on the bfs_encryption setting, like this:
-    # bfs_protocol = "https" if str_to_bool(conf, CKey.bfs_encryption, True)
-    # else "http"
-    bfs_protocol = "http"
+    bfs_protocol = "https" if str_to_bool(conf, CKey.bfs_encryption, True) else "http"
```

Comment:
Ok, I see this one is already useful, without the new bucketfs-python version, so we should merge it, also if we haven't released bucketfs-python